plugin.video.srgssr_ch_replay/addon.xml

```diff
@@ -39,4 +39,4 @@ Um die Sendungen streamen zu können, muss ein Konto bei der SRG SSR erstellt we
     <website>https://www.srgssr.ch</website>
     <forum>https://github.com/ManBehindMooN/kodi_plugin_video_srgssr_ch_replay/discussions</forum>
   </extension>
-</addon>
\ No newline at end of file
+</addon>

```

Comment:
The French translation for 
* <summary lang="fr_FR"> 
* <summary lang="fr_FR">
*  <disclaimer lang="fr_FR"> 

Should be included into the addon.xml

Reply:
Oh forgot them! I'll also add them ;)